### PR TITLE
Matthew g

### PR DESCRIPTION
**Pull request by MatthewGan923**

### CODE REVIEW

File: example/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package io.flutter.plugins;
```
Comment (firejake308) [inline example/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java]: What is the `example` directory? It looks like it doesn't exist in master

File: lib/main.dart [hunk 2]
```diff
@@ -14,6 +15,7 @@ void main() {
     ["BMI > 30", 5, Icons.warning],
     ["Reoperation", 7, Icons.replay],
     ["Prosthetic reconstruction", 1, Icons.content_cut],
+    ["Emergency", 1, Icons.error]
```
Comment (firejake308) [inline lib/main.dart]: Can you also go ahead and update the numbers for the other risk factors? Here's the new numbers from Dr Rahimi, for your reference

- Female = 4
- BMI>=28 = 9
- Smoker/Former smoker = 2
- Redo groin = 13
- Prosthesis = 7

Also, the icons for emergency and BMI are pretty similar. Can you try replacing the BMI icon with `Icons.accessibility_new`?

File: lib/main.dart [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (firejake308) [inline lib/main.dart]: Can you delete this empty line at the top? I know it's kinda nitpicky, but I like having the imports at the very top of the file

**Review by firejake308 — CHANGES_REQUESTED** [review]
I see there's a merge conflict, and I can take care of that since I wrote the conflicting code. Besides that, I had a question about the new `example` directory, and I'd also appreciate if you could update the numbers to match the revised algorithm.

File: lib/main.dart (hunk 2 above)
Comment (MatthewGan923) [reply lib/main.dart]: Just fixed

File: lib/main.dart (hunk 3 above)
Comment (MatthewGan923) [reply lib/main.dart]: Just fixed

File: example/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java (hunk 1 above)
Comment (MatthewGan923) [reply example/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java]: Not sure where this come from at all. just delted and it seems to be running fine.

**Review by firejake308 — APPROVED** [review]

**Comment by firejake308** [comment]
@MatthewGan923 Sweet, this is ready to go. I'll go ahead and merge it into master

**Pull request merged**